iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -172,10 +172,10 @@ cluster_name=defaultCluster
 # Datatype: boolean
 # enable_mem_control=true
 
-# Memory Allocation Ratio: StorageEngine, QueryEngine, SchemaEngine, Consensus and Free Memory.
-# The parameter form is a:b:c:d:e, where a, b, c, d and e are integers. for example: 1:1:1:1:1 , 6:2:1:1:1
-# If you have high level of writing pressure and low level of reading pressure, please adjust it to for example 6:1:1:1:2
-# storage_query_schema_consensus_free_memory_proportion=3:3:1:1:2
+# Memory Allocation Ratio: StorageEngine, QueryEngine, SchemaEngine, StreamingEngine, Consensus and Free Memory.
+# The parameter form is a:b:c:d:e:f, where a, b, c, d, e and f are integers. for example: 1:1:1:1:1:1 , 6:2:1:1:1:1
+# If you have high level of writing pressure and low level of reading pressure, please adjust it to for example 6:1:1:1:1:2
```

Comment:
```suggestion
# Memory Allocation Ratio: StorageEngine, QueryEngine, SchemaEngine, StreamingEngine, Consensus and Free Memory.
# The parameter form is a:b:c:d:e:f, where a, b, c, d, e and f are integers. for example: 1:1:1:1:1:1 , 6:2:1:1:1:1
# If you have high level of writing pressure and low level of reading pressure, please adjust it to for example 6:1:1:1:1:1
```

THIS WILL BE FIXED IN PR: https://github.com/apache/iotdb/pull/11424